addon/templates/project/form.hbs

```diff
@@ -80,6 +80,23 @@
         }}
       />
 
+      <div uk-alert class="uk-flex uk-flex-middle uk-form-label">
+        <span>
+          <div class="uk-text-bold">
+            {{t "ember-gwr.specialConstructionWarning.header"}}
+          </div>
+          <div>
+            {{t "ember-gwr.specialConstructionWarning.1"}}
```

Comment:
The .1 and .2 seem kinda weird, but I can't think of another way either.

Reply:
Maybe part1 and part2 ? I was expecting an option like with the other code labels.

Reply:
I changed it to `part1` and `part2`.